packages/patrol/example/ios/RunnerTests/RunnerTests.swift

```diff
@@ -39,4 +37,12 @@ final class RunnerTests: XCTestCase {
     let result = PatrolUtils.createMethodName(fromPatrolGeneratedGroup: input)
     XCTAssertEqual(result, expectedOutput)
   }
+
+  func testCreateMethodNameFromPatrolGeneratedGroup_ExampleWithGroups() {
+    let input = "example_test at the beginning"
+    let expectedOutput = "exampleTestatthebeginning"  // FIXME: This is temporary
+    let result = PatrolUtils.createMethodName(fromPatrolGeneratedGroup: input)
+    NSLog(result)
```

Comment:
It's OK for this FIXME to land in a `dev` release on `develop` branch.
I'll get to it once we'll be stabilizing before merging to `master`.